src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -2236,6 +2237,7 @@ MergeTreeData::MutableDataPartPtr StorageReplicatedMergeTree::attachPartHelperFo
 
         try
         {
+            Expect404ResponseScope scope; // 404 is not an error
             part->loadChecksums(true);
```

Comment:
If we choose to drag flag over the whole stack, than it would be on all funtions.
Here we expect race condition, we do not what to alert when it happened.